backend/src/rtp.rs

```diff
@@ -1676,7 +1733,9 @@ impl Endpoint {
                     "Invalid RTP: no seqnum in payload of RTX packet; payload len: {}",
                     incoming.payload_range_in_header.len()
                 );
-                return None;
+                return Err(EndpointError::MaxReceivedTccSeqnumError(
+                    "no seqnum in payload of RTX packet".into(),
+                ));
```

Comment:
I don't think there is any value in passing a string here, just a parameterless error should be fine.

Also, I think the name should be more like `MissingRTXSeqNum` or something.